lib/services/plugins-service.ts

```diff
@@ -185,6 +185,22 @@ export class PluginsService implements IPluginsService {
 		return this.executeForAllInstalledPlatforms(action);
 	}
 
+	public installDevDependencies(): IFuture<void> {
+		return (() => {
+			let installedDependencies = this.$fs.exists(this.nodeModulesPath).wait() ? this.$fs.readDirectory(this.nodeModulesPath).wait() : [];
+			let packageJsonContent = this.$fs.readJson(this.getPackageJsonFilePath()).wait();
+			let devDependencies = _.keys(packageJsonContent.devDependencies);
+			if(devDependencies.length && (this.$options.force || _.difference(devDependencies, installedDependencies).length)) {
+				let command = `npm install ${devDependencies.join(" ")}`;
```

Comment:
nope, I want only devDependencies to be installed, while `npm i --development` is installing everything **including** devDependencies.